Libraries/process/Sources/DefaultContext.cpp

```diff
@@ -357,6 +386,16 @@ shellExpand(std::string const &s) const
     }
     expandedString = WideStringToString(buffer);
 
+#else
+#ifdef __OpenBSD__
```

Comment:
You can just make this `#elif __OpenBSD__` and drop the extra `#endif` at line 408.

Reply:
y